workspaces/arborist/test/arborist/audit.js

```diff
@@ -29,7 +35,9 @@ t.test('audit finds the bad deps', async t => {
 
 t.test('no package lock finds no bad deps', async t => {
   const path = resolve(fixtures, 'deprecated-dep')
-  t.teardown(auditResponse(resolve(fixtures, 'audit-nyc-mkdirp/audit.json')))
```

Comment:
Should we generally avoid path strings with slashes in tests, and use `resolve` instead to traverse the path?